### PR TITLE
Fix relative paths in deposit script

### PR DESCRIPTION
**Pull request by fedgiac**
Fixes issue #78 and makes the behavior of the deposit script consistent with that of the withdraw script after PR #155. With this change, the command-line examples in the readme become correct.

### CODE REVIEW

File: scripts/transfer_approve_deposit.js [hunk 1]
```diff
@@ -18,8 +20,8 @@ module.exports = async (callback) => {
     const GnosisSafe = artifacts.require("GnosisSafe")
     const masterSafe = await GnosisSafe.at(argv.masterSafe)
 
-    const deposits = require(argv.depositFile)
-    // TODO - make a simpler to construct deposit file style
+    const deposits = JSON.parse(await fs.readFile(argv.depositFile, "utf8"))
+    // TODO - make it simpler to construct deposit file style
```
Comment (bh2smith) [inline scripts/transfer_approve_deposit.js:24]: I think this TODO should be removed if we aren't going to TODO it.

File: scripts/transfer_approve_deposit.js [hunk 2]
```diff
@@ -18,8 +20,8 @@ module.exports = async (callback) => {
     const GnosisSafe = artifacts.require("GnosisSafe")
     const masterSafe = await GnosisSafe.at(argv.masterSafe)
 
-    const deposits = require(argv.depositFile)
-    // TODO - make a simpler to construct deposit file style
+    const deposits = JSON.parse(await fs.readFile(argv.depositFile, "utf8"))
```
Comment (bh2smith) [inline scripts/transfer_approve_deposit.js:23]: I am struggling to understand why this change is necessary.

Comment (fedgiac) [reply scripts/transfer_approve_deposit.js:23]: The difference is that `require` uses the path of the script as the base path, while `fs.readFile` uses the current working directory.
So, if your current working directory is `/path/to/dex-liquidity-provision/` and your deposit file is in `./examples/deposits.json`, then before this PR you'd have to run:
```
npx truffle exec scripts/transfer_approve_deposits.js [...] --depositFile="../examples/deposits.json"
```
(Note the two dots in the relative path.)
Now you can just use:

``` 
npx truffle exec scripts/transfer_approve_deposits.js [...] --depositFile="./examples/deposits.json"
```
(Note the single dots in the relative path.)
I noticed this bug when I tried to rely on the autocomplete to fill the path, which doesn't work because it autocompleted based on the current working directory.

File: scripts/transfer_approve_deposit.js (hunk 1 above)
Comment (fedgiac) [reply scripts/transfer_approve_deposit.js:24]: Fair enough, I don't see this happening any time soon.

File: scripts/transfer_approve_deposit.js (hunk 2 above)
Comment (bh2smith) [reply scripts/transfer_approve_deposit.js:23]: Thanks for clearing that up! I now understand.
